go.mod

```diff
@@ -213,7 +213,6 @@ require (
 	github.com/cespare/xxhash/v2 v2.1.2 // indirect
 	github.com/chai2010/gettext-go v1.0.2 // indirect
 	github.com/coreos/go-systemd/v22 v22.3.3-0.20220203105225-a9a7ef127534 // indirect
-	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f // indirect
```

Comment:
revert

Reply:
I can't find coreos on imports or the dependency graph. Do you remember why we need it?

Let's import it in the correct place, if it is indeed legit.

Reply:
this is the one that get's removed on mine machine and dependabot, but is required.
```
go mod graph | grep coreos/pkg
github.com/gravitational/teleport github.com/coreos/pkg@v0.0.0-20180928190104-399ea9e2e55f
github.com/spf13/viper@v1.4.0 github.com/coreos/pkg@v0.0.0-20180928190104-399ea9e2e55f
go.etcd.io/etcd@v0.0.0-20191023171146-3cf2f69b5738 github.com/coreos/pkg@v0.0.0-20160727233714-3ac0863d7acf
```